.yamato/upm-ci.yml

```diff
@@ -8,10 +8,14 @@ test_platforms:
     type: Unity::VM
     image: package-ci/win10:stable
     flavor: b1.large
+    utr_install: curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output utr.bat
+    utr_performance_test: utr --suite=editor --clean-library --category="Performance" --testproject=TestProject/Usd-Development-Performance --editor-location=.Editor --report-performance-data
```

Comment:
To report data to Observer I think you also need to pass a project ID, ie: `--performance-project-id=com.unity.formats.usd`